runtime-diffs/moonbase/3400.txt

```diff
@@ -0,0 +1,57 @@
+Pallets:
+    ~ MoonbeamLazyMigrations
+        Calls:
+            + approve_assets_to_migrate
+            - clear_suicided_storage
+            + finish_foreign_assets_migration
+            + migrate_foreign_asset_approvals
+            + migrate_foreign_asset_balances
+            + start_foreign_assets_migration
+        Storage Entries:
+            + ApprovedForeignAssets
+            + ForeignAssetMigrationStatusValue
+            - SuicidedContractsRemoved
+    ~ Multisig
+        Calls:
+            ~ as_multi
+            ~ as_multi_threshold_1
+    ~ OpenTechCommitteeCollective
+        Calls:
+            ~ execute
+            ~ propose
+        Storage Entries:
+            ~ ProposalOf (Changed: )
```

Comment:
This was changed in runtime 3300, the diff is not expected for runtime 3400.

Reply:
Never mind, the differences are exrpected because the `RuntimeCall` will change every time an extrinsic is modified.